src/Lifecycle.js

```diff
@@ -229,14 +230,12 @@ function _handleRestoreFailure(e) {
 
     let msg = e.message;
     if (msg == "OLM.BAD_LEGACY_ACCOUNT_PICKLE") {
-        msg = "You need to log back in to generate end-to-end encryption keys "
-            + "for this device and submit the public key to your homeserver. "
-            + "This is a once off; sorry for the inconvenience.";
+        msg = _t('You need to log back in to generate end-to-end encryption keys for this device and submit the public key to your homeserver. This is a once off; sorry for the inconvenience') + '.';
```

Comment:
it'd be nice to keep long messages like this split over multiple lines if possible for legibility (and per the codestyle)